tests/admin_views/models.py

```diff
@@ -957,3 +957,7 @@ def __str__(self):
 
 class RelatedWithUUIDPKModel(models.Model):
     parent = models.ForeignKey(ParentWithUUIDPK, on_delete=models.SET_NULL, null=True, blank=True)
+
+
+class Animal(models.Model):
```

Comment:
A new model and modeladmin isn't needed -- use an existing one that's already tested in `test_actions.py`.